src/ansys/openapi/common/_base/_types.py

```diff
@@ -110,12 +110,23 @@ def call_api(
         header_params: Union[Dict[str, Union[str, int]], None] = None,
         body: Optional[DeserializedType] = None,
         post_params: Optional[List[Tuple[str, Union[str, bytes]]]] = None,
-        files: Optional[Dict[str, str]] = None,
+        files: Optional[Mapping[str, str]] = None,
         response_type: Optional[str] = None,
         _return_http_data_only: Optional[bool] = None,
         collection_formats: Optional[Dict[str, str]] = None,
         _preload_content: bool = True,
         _request_timeout: Union[float, Tuple[float, float], None] = None,
-        response_type_map: Optional[Dict[int, Union[str, None]]] = None,
+        response_type_map: Optional[Mapping[int, Union[str, None]]] = None,
     ) -> Union[requests.Response, DeserializedType, None]:
         """Provide method signature for calling the API."""
+
+
+class _Unset:
+    @staticmethod
+    def __bool__() -> Literal[False]:
+        return False
+
+
+Unset_Type = _Unset
```

Comment:
This is absurd, but I couldn't find a way to get a type available to the code template any other way. It's technically still possible to instantiate a new Unset object, i.e. `Unset_Type()`, but the name makes it less obvious that that's what you could do.

Reply:
Would a TypeVar bound to _Unset help here?

Reply:
Still can't get it to work, I think the problem is that using the `bound` parameter expects it to match a subtype, and you also can't make a TypeVar 'exactly' one type, it can only be 'exactly' two or more types.